### PR TITLE
Add an example of using Pulumi with Ansible

### PR DESCRIPTION
**Pull request by joeduffy**
This change adds an example of deploying Wordpress to AWS EC2 using
Pulumi and Ansible together. Pulumi handles provisioning the AWS
infrastructure required to run Wordpress in AWS, including the EC2
instance and an RDS MySQL database, all running within a VPC with
proper public and private subnets, exposed to the Internet using an
Elastic IP address. Ansible handles configuring the EC2 virtual
machine after it's been provisioned with a playbook that knows how
to install and configure Wordpress. The entire deployment is
orchestrated by a single `pulumi up` thanks to our command package.

### CODE REVIEW

File: aws-yaml-ansible-wordpress/Pulumi.yaml
Comment (jkodroff) [inline aws-yaml-ansible-wordpress/Pulumi.yaml:53]: Suggest using AWSX 1.0 for this if it works with YAML (I forget if it does).

**Review by jkodroff — APPROVED** [review]
All in all, this looks good.

I'm not the foremost Anisible expert, so I might wait on a second approval for additional comments on how this code might be improved.

File: aws-yaml-ansible-wordpress/Pulumi.yaml
Comment (joeduffy) [reply aws-yaml-ansible-wordpress/Pulumi.yaml:61]: Thanks Josh. I added a note about the fact that you can choose to use AWSX. Especially for the Ansible audience, I actually thought it would be best to lay out all the raw building blocks so there's a bit less "magic" to the example.

**Comment by joeduffy** [comment]
The Airflow example is failing. This seems to happen from time to time. @stack72 Is this test just flaky? I'll merge anyway but it definitely makes me feel like I'm not doing the right thing 😦

**Pull request merged**
